iguana/3rd_party

```diff
@@ -13,68 +13,68 @@
-        {"node9_EU": "0392e4c9400e69f28c6b9e89d586da69d5a6af7702f1045eaa6ebc1996f0496e1f" },
-        {"smdmitry_AR": "0397b7584cb29717b721c0c587d4462477efc1f36a56921f133c9d17b0cd7f278a" },
-        {"nodeone_NA": "0310a249c6c2dcc29f2135715138a9ddb8e01c0eab701cbd0b96d9cec660dbdc58" },
-        {"gcharang_SH": "02a654037d12cdd609f4fad48e15ec54538e03f61fdae1acb855f16ebacac6bd73" },
-        {"cipi_EU": "026f4f66385daaf8313ef30ffe4988e7db497132682dca185a70763d93e1417d9d" },
-        {"etszombi_AR": "03bfcbca83f11e622fa4eed9a1fa25dba377981ea3b22e3d0a4015f9a932af9272" },
-        {"pbca26_NA": "03c18431bb6bc95672f640f19998a196becd2851d5dcba4795fe8d85b7d77eab81" },
-        {"mylo_SH": "026d5f29d09ff3f33e14db4811606249b2438c6bcf964876714f81d1f2d952acde" },
-        {"swisscertifiers_EU": "02e7722ebba9f8b5ebfb4e87d4fa58cc75aef677535b9cfc060c7d9471aacd9c9e" },
-        {"marmarachain_AR": "028690ca1e3afdf8a38b421f6a41f5ff407afc96d5a7a6a488330aae26c8b086bb" },
-        {"karasugoi_NA": "02f803e6f159824a181cc5d709f3d1e7ff65f19e1899920724aeb4e3d2d869f911" },
-        {"phm87_SH": "03889a10f9df2caef57220628515693cf25316fe1b0693b0241419e75d0d0e66ed" },
-        {"oszy_EU": "03c53bd421de4a29ce68c8cc83f802e1181e77c08f8f16684490d61452ea8d023a" },
-        {"chmex_AR": "030cd487e10fbf142e0e8d582e702ecb775f378569c3cb5acd0ff97b6b12803588" },
-        {"dragonhound_NA": "029912212d370ee0fb4d38eefd8bfcd8ab04e2c3b0354020789c29ddf2a35c72d6" },
-        {"strob_SH": "0213751a1c59d3489ca85b3d62a3d606dcef7f0428aa021c1978ea16fb38a2fad6" },
-        {"madmax_EU": "0397ec3a4ad84b3009566d260c89f1c4404e86e5d044964747c9371277e38f5995" },
-        {"dudezmobi_AR": "033c121d3f8d450174674a73f3b7f140b2717a7d51ea19ee597e2e8e8f9d5ed87f" },
-        {"daemonfox_NA": "023c7584b1006d4a62a4b4c9c1ede390a3789316547897d5ed49ff9385a3acb411" },
-        {"nutellalicka_SH": "0284c4d3cb97dd8a32d10fb32b1855ae18cf845dad542e3b8937ca0e998fb54ecc" },
-        {"starfleet_EU": "03c6e047218f34644ccba67e317b9da5d28e68bbbb6b9973aef1281d2bafa46496" },
-        {"mrlynch_AR": "03e67440141f53a08684c329ebc852b018e41f905da88e52aa4a6dc5aa4b12447a" },
-        {"greer_NA": "0262da6aaa0b295b8e2f120035924758a4a630f899316dc63ee15ef03e9b7b2b23" },
-        {"mcrypt_SH": "027a4ca7b11d3456ff558c08bb04483a89c7f383448461fd0b6b3b07424aabe9a4" },
-        {"decker_EU": "027777775b89ff548c3be54fb0c9455437d87f38bfce83bdef113899881b219c9e" },
-        {"dappvader_SH": "025199bc04bcb8a17976d9fe8bc87763a6150c2727321aa59bf34a2b49f2f3a0ce" },
-        {"alright_DEV": "03b6f9493658bdd102503585a08ae642b49d6a68fb69ac3626f9737cd7581abdfa" },
-        {"artemii235_DEV": "037a20916d2e9ea575300ac9d729507c23a606b9a200c8e913d7c9832f912a1fa7" },
-        {"tonyl_DEV": "0258b77d7dcfc6c2628b0b6b438951a6e74201fb2cd180a795e4c37fcf8e78a678" },
-        {"decker_DEV": "02fca8ee50e49f480de275745618db7b0b3680b0bdcce7dcae7d2e0fd5c3345744" }
+	{"alrighttt_DEV", "02a876c6c35060041f6beadb201f4dfc567e80eedd3a4206ff10d99878087bd440"}, // 0
```

Comment:
> "alrighttt_DEV", "02a876c6c35060041f6beadb201f4dfc567e80eedd3a4206ff10d99878087bd440"

comma has to be replaced with colon

`"alrighttt_DEV": "02a876c6c35060041f6beadb201f4dfc567e80eedd3a4206ff10d99878087bd440"`

Reply:
for all